packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -296,10 +296,10 @@ export async function getBlock<ReturnFormat extends DataFormat>(
 		const result = {
 			...res,
 			transactions: res.transactions ?? [],
-		}
+		};
```

Comment:
Please try to not format existing code

Reply:
Gotcha. format-on-save guilty once again :)